src/frontend/src/app/services-health/services-health.component.ts

```diff
@@ -28,15 +28,15 @@ export class ServicesHealthComponent implements OnInit {
       .subscribe(
       (data) => {
         if (data.error) {
+          /** request completed */
+          this.ngProgress.done();
```

Comment:
this is creating problem!

Reply:
and no change needed in `list-tasks.component.html`?

Reply:
No, there is no need to change in `list-tasks.component.html`, because progress bar appears on the top of the page in each case.